docs/content.md

```diff
@@ -535,6 +535,98 @@ view = api.content.get_view(
 %
 % self.assertEqual(view.__name__, u'plone')
 
+(content-get-parents-example)=
+
+## Get parent objects
+
+You can get all parents of a content object (from immediate parent to the portal root) using the {meth}`api.content.get_parents` method.
+
+```python
+from plone import api
+portal = api.portal.get()
+team = portal['about']['team']
+
+# Get all parents
+parents = api.content.get_parents(obj=team)
+```
+
+% invisible-code-block: python
+%
+% self.assertEqual(len(parents), 2)
```

Comment:
```suggestion
% self.assertEqual(len(parents), 2)
```
I already mentioned that it would be better for this one to be a `self.assertListEqual`

Reply:
pushing with newer changes inna while!